lib/rdstation/client.rb

```diff
@@ -32,6 +33,7 @@ def initialize(rdstation_token, auth_token, identifier="integração")
     # "Detalhes do Lead".
     #
     def create_lead(lead_hash)
+      warn "DEPRECATION WARNING: create_lead is deprecated."
```

Comment:
Se aplica a todos os outros. Acho que vale continuar a frase e informar quando vai parar de funcionar:

`DEPRECATION WARNING: create_lead is deprecated and will be removed in 2.0.0`